cmd/nirvana/project/init.go

```diff
@@ -744,61 +796,101 @@ OUTPUT_DIR := ./bin
 # Build direcotory.
 BUILD_DIR := ./build
 
-# Git commit sha.
-COMMIT := $(strip $(shell git rev-parse --short HEAD 2>/dev/null))
-COMMIT := $(COMMIT)$(shell git diff-files --quiet || echo '-dirty')
-COMMIT := $(if $(COMMIT),$(COMMIT),"Unknown")
-{{ if .GoMod }}
-# This will force go to use the vendor files instead of using the $GOPATH/pkg/mod. (vendor mode)
-# more info: https://github.com/golang/go/wiki/Modules#how-do-i-use-vendoring-with-modules-is-vendoring-going-away
-export GOFLAGS := -mod=vendor
-{{ end }}
+# Current version of the project.
```

Comment:
The `Makefile`  in golang-template-project has been updated, prefer to update to the latest.
As `nirvana init` will be used for some new projects in v3.0.

Reply:
I'll update the Makefile except for the version part since go-common is private and nirvana is an open-source project